pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -84,15 +84,15 @@ sub _check_for_apache_chroot {
             {
                 'key'        => 'Apache_vhosts_not_segmented',
                 'type'       => $Cpanel::Security::Advisor::ADVISE_BAD,
-                'text'       => ['Apache vhosts are not segmented or chroot()ed.'],
-                'suggestion' => [
+                'text'       => $self->_lh->maketext('Apache vhosts are not segmented or chroot()ed.'),
+                'suggestion' => $self->_lh->maketext(
```

Comment:
Are these multi-line formatting options still going to harvest correctly?  This looks like a potential deviant to me.